client/my-sites/email/email-management/home/email-mailbox-action-menu.jsx

```diff
@@ -61,11 +65,14 @@ const removeEmailForwardMailbox = ( { dispatch, mailbox } ) => {
  * Returns the available menu items for Titan Emails
  *
  * @param {Object} titanMenuParams The argument for this function.
- * @param {string} titanMenuParams.email The email address of the Titan account
- * @param {Function} titanMenuParams.translate The translate function
+ * @param {Function} titanMenuParams.showRemoveMailboxDialog The function that removes modal dialogs for confirming mailbox removals
+ * @param {Object} titanMenuParams.mailbox The mailbox object.
+ * @param {Function} titanMenuParams.translate The translate function.
  * @returns Array of menu items
  */
-const getTitanMenuItems = ( { email, translate } ) => {
+const getTitanMenuItems = ( { showRemoveMailboxDialog, mailbox, translate } ) => {
```

Comment:
Nit: these arguments aren't in alphabetical order.
```suggestion
const getTitanMenuItems = ( { mailbox, showRemoveMailboxDialog, translate } ) => {
```